src/ring_theory/dedekind_domain.lean

```diff
@@ -249,4 +255,107 @@ begin
                  fractional_ideal.map_mul, fractional_ideal.map_div, inv_eq] },
 end
 
+lemma is_dedekind_domain_inv.mul_inv_eq_one [algebra A K] [is_fraction_ring A K]
```

Comment:
I think it makes sense to start a namespace and some variables. There's a lot of repetition in the next few statements.

Reply:
Done. I also moved `is_field.to_principal_ideal_domain` out of the new namespace, to a new place up the import hierarchy.